src/index.html

```diff
@@ -10,23 +10,37 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar airbnb</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="big-search"
+      data-qa="big"
+    >
+      <input
+        class="input input__big"
```

Comment:
Your usage of `input__big` and `input__small` look very similar to a modifier; maybe you meant to name them `input--big` and `input--small`?